content/components/toggle-switch.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Toggle switch
+description: A toggle switch is used to immediately toggle a setting on or off.
+---
+
+import {Box, Text} from '@primer/components'
+
+## Anatomy
+
+<Box display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+    <Box as="ul" m="0" p="0">
+        <li><strong>Track</strong>: The clickable area where the toggle switch moves back and forth</li>
+        <li><strong>Knob</strong>: Changes color and position depending on whether the switch is on or off</li>
+        <li><strong>Status</strong>: A text label showing whether the switch is on or off</li>
+    </Box>
+    <img
+        width="456"
+        role="presentation"
+        src="https://user-images.githubusercontent.com/2313998/159587114-97d7de3c-9ff6-4f13-beca-6e4da2d548c6.png"
+    />
+</Box>
+
+## Contextual content
+
+<Box mb={5} display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
+    <div>
+    <Box as="p" mt="0">Every toggle switch should have a label that says what is being turned on or off. Optionally, you may include a description about the setting that is being turned on or off.</Box>
+    <Box as="p" mt="0">Labels should be succinct, and can describe a object or an action (for example: "Discussions" or "Automatically watch repositories"). Don't use phrases that describe the switch's state in your label.</Box>
+    </div>
+    <img
+        width="456"
+        role="presentation"
+        src="https://user-images.githubusercontent.com/2313998/159587115-5ffcdefb-f5e9-4180-8dc3-5328b0440f7f.png"
+    />
+</Box>
+
+
+
+<DoDontContainer>
+  <Do>
+    <img role="presentation" src="https://user-images.githubusercontent.com/2313998/159587088-99e8fe4c-59a3-4452-836d-bd6f2297045a.png" />
+    <Caption>Use succinct phrases with verbs or nouns</Caption>
+  </Do>
+  <Dont>
+    <img role="presentation" src="https://user-images.githubusercontent.com/2313998/159587096-5f8a6913-df95-4ea0-9838-a366faf95e52.png" />
+    <Caption>Don't use phrases that describe the switch's state</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Representing mixed values
+
+Since a toggle switch can only be "on" or "off", it could be confusing to use a switch to represent mixed values.
+
+If you have a section of configuration options that can be disabled, use a button instead of a toggle switch. Disabling the section turns off all configuration options, but remembers their previous state when the user wants to turn the section back on. While the section is disabled, the configuration options are hidden.
```

Comment:
```suggestion
If you have a group of configuration options that can be disabled, use a button instead of a toggle switch. Disabling the group turns off all configuration options, but remembers their previous state when the user wants to turn that section back on. While the section is disabled, the configuration options are hidden.
```